DOCUMENTATION.md

```diff
@@ -3492,22 +3492,15 @@ This method registers a new api key for the current user with the name given.
 
 **Example**  
 ```js
-resin.models.apiKey.create(apiKeyName).then(function(apiKey) {
-	console.log(apiKey);
-});
+resin.models.apiKey.create(apiKeyName).then(function(apiKey) {
	console.log(apiKey);
});
```

Comment:
These `^M` characters are because of line ending issues, which I think means you're running on Windows and you need extra git settings. Looks like you need to follow this: https://help.github.com/articles/dealing-with-line-endings/#platform-windows, which means running:

```
git config --global core.autocrlf true
```

Then if you readd this file, this diff should go away.